### ISSUE TITLE
TSLA p15838coll23 missing rights statements

### ISSUE THREAD

**Issue by markpbaggett** [DigitalLibraryofTennessee/DLTN_XSLT#35]
## Why are You Reporting an Issue

- [ ] Add New Set(s)
- [ ] Reingest Existing Set
- [x] Report Problem with an Existing Set or XSL transform

## Problem with Set or Transform

This is a subticket of [Issue 31](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/31).

Currently, all of these records that are going to DPLA are having a rights statement automatically applied; this happened when we first went to DPLA, and we've never had time to fix this. Luckily, the rights statement being applied is pretty generic and is taken from TSLA. We'd like to work on this for August, but some of this could potentially be addressed by TSLA. Let me know what you think is doable, and we can come up with a plan for summer.

Potential ideas / solutions:

1. We drop the set altogether.
2. TSLA fixes these in ContentDM and we reingest.
3. UTK drops the rights transform and we fix by harvesting into BaseX and running a transform to determine if it has rights before it goes into our Aggregator.

Here are the records missing rights from this set:

**No list here.** There are 3988 records from p15838coll23 missing rights. Should we just remove this set altogether?

**Comment by markpbaggett** [comment]
@jrandlestsla, interesting!  The truth is that our aggregation process is agnostic to metadata only records.  Basically, if it has a OAI record, we grab it and map it to MODS.  Then, we programmatically build out the thumbnail link.  

So what to do here is interesting.  The most convenient thing for me would be to only have the objects with metadata in this set / feed.  If not, we have to have a test, and we have to do that test external to Repox. Right now, it there is no rights, we just give them all rights.

Just curious:  what is you guys' stance in having the metadata only records in a separate collection / set?  How much work would that be?  I have no knowledge of ContentDM, so if this is a lot of work or if this is a bad idea for another reason for you guys, I understand. 

I'm going to mention @CanOfBees in case we need to brainstorm what to do here.  I'd really like to keep the 160+ good ones if we could. 

Also, would you potentially be adding more objects with metadata records to this one? I think that'd help us think about what to do.

**Comment by jrandlestsla** [comment]
The metadata-only records and the records with images need to stay in the same collection, as they represent one physical collection. Also, the metadata-only records are being updated as the maps are scanned, so eventually all the items in the collection will have images. I don't think that is a good idea for that reason among others. 

It's basically a collection in progress, that will take years to complete. Is there any way to use a query to specify a set, meaning only those currently with images, and send that to DPLA?

**Comment by markpbaggett** [comment]
@jrandlestsla, yes, it's possible.  it's similar to what we do for CMHF, so I think we can do that for this one as well.  can you give me an example of one that is metadata only and one that is digitized? that will help us figure out what we need to write.  

thanks again for working on all of this!

**Comment by jrandlestsla** [comment]
Great, Mark! 

This one has an image: http://teva.contentdm.oclc.org/cdm/singleitem/collection/p15138coll23/id/384/rec/2

This one is metadata-only: http://teva.contentdm.oclc.org/cdm/singleitem/collection/p15138coll23/id/7299/rec/4414

**Comment by CanOfBees** [comment]
@jrandlestsla - thanks for those examples!

**Comment by markpbaggett** [comment]
@jrandlestsla, thanks for sending.  Here these are as OAI for reference:

http://cdm15138.contentdm.oclc.org/oai/oai.php?verb=GetRecord&identifier=oai:cdm15138.contentdm.oclc.org:p15138coll23/384&metadataPrefix=oai_dc

http://cdm15138.contentdm.oclc.org/oai/oai.php?verb=GetRecord&identifier=oai:cdm15138.contentdm.oclc.org:p15138coll23/7299&metadataPrefix=oai_dc
